### PR TITLE
More tests

### PR DESCRIPTION
**Pull request by mnapoli**
This time I test that we reuse the compiled container if it already exists.

I haven't added any tests yet around other kind of caches.

### CODE REVIEW

File: tests/Functional/FunctionalTest.php [hunk 1]
```diff
@@ -72,8 +89,13 @@ private function runSymfonyConsole(): Process
...
         ]);
-        $dockerCommand->mustRun();
+        $dockerCommand->run();
 
         return $dockerCommand;
     }
+
+    private function assertCommandIsSuccessful(Process $command): void
+    {
+        $this->assertTrue($command->isSuccessful(), $command->getOutput() . PHP_EOL . $command->getErrorOutput());
```
Comment (Nyholm) [inline tests/Functional/FunctionalTest.php:99]: I did something similar in #24, But I also made sure that the error output was empty.

**Comment by Nyholm** [comment]
I'll merge this an rebase my PR.

**Comment by Nyholm** [comment]
Thank you

**Comment by mnapoli** [comment]
Woops, please don't rebase, I'm working on PRs on top of one another ^^

**Pull request merged**